CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## CHANGELOG
 
+### 2.0.9
+
+* Update Public API requests to POST
```

Comment:
I see value in adding a note here, mentioning explicitly we don't do GET requests anymore. 
```suggestion
* Deprecation of GET requests on Public API in favor of POST requests. 
```